tests/kernel/kdump_over_nfs.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Configure and run kdump over NFS.
+# The test is configuring - on the nfs client side - the kdump with
+# KDUMP_SAVEDIR set to NFS share. Then the actual crash is triggered,
+# still on the NFS client side and once the reboot happens, the nfs share,
+# set as the target in the KDUMP_SAVEDIR, is mounted and the crash files
+# are checked
+#
+# Maintainer: QE Kernel <kernel-qa@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use kdump_utils;
+use lockapi;
+use power_action_utils 'power_action';
+
+sub run {
+    my ($self) = @_;
+    my $role = get_required_var('ROLE');
+    my $kdump_export = get_required_var('KDUMP_SAVEDIR');
```

Comment:
We don't need `$kdump_export` anymore, but we could die on missing variable.

Reply:
ah right. I missed that. thx

I left this:
get_required_var('KDUMP_SAVEDIR');

this will die under the hood with meaningful info

You think this is good enough?